CruxPay/Classes/CruxJSBridge.swift

```
@@ -0,0 +1,95 @@
+        let promiseDeps = try! String(contentsOfFile: promiseDepsPath)
+        _ = context?.evaluateScript("var console = {log: function(message) { _consoleLog(message) }, warn: function(message) { _consoleLog(message) } }")
+        
+        let consoleLog: @convention(block) (String) -> Void = { message in
+            print("console.log: " + message)
+        }
+        context?.setObject(unsafeBitCast(consoleLog, to: AnyObject.self),
+           forKeyedSubscript: "_consoleLog" as NSCopying & NSObjectProtocol)
+        _ = context?.evaluateScript("var window = this;")
+        _ = context?.evaluateScript(requestDeps)
+        _ = context?.evaluateScript(promiseDeps)
+        JSFetch.provideToContext(context: context!, hostURL: "https://www.cruxpay.com")
+        JSIntervals.provideToContext(context: context!)
+        
+        let cruxJS = try! String(contentsOfFile: cruxJSPath)
+        _ = context?.evaluateScript("var crypto = { getRandomValues: function(bytes) { return _getRandomValues(bytes) } }")
+        let getRandomValues: @convention(block) (Array<UInt8>) -> Array<UInt8> = { arr in
+            let count = arr.count
+            return AES.randomIV(count)
+        }
+        context?.setObject(unsafeBitCast(getRandomValues, to: AnyObject.self),
+                                 forKeyedSubscript: "_getRandomValues" as (NSCopying & NSObjectProtocol)?)
+        _ = context?.evaluateScript(cruxJS)
+        
+        return context
+    }()
+    
+    init(configBuilder: CruxClientInitConfig.Builder) {
+        prepareCruxClientInitConfig(configBuilder: configBuilder)
+        context?.evaluateScript("cruxClient = new window.CruxPay.CruxClient(cruxClientInitConfig)")
+        context?.evaluateScript("""
+            cruxClient.init()
+            .then(() => {
+                console.log('CruxClient initialized');
+            }).catch((err) => {
+                console.log('CruxClient error', err);
```

error is being suppressed here.